lsp-types/src/Language/LSP/Types/WorkspaceEdit.hs

```diff
@@ -359,6 +359,11 @@ data ApplyWorkspaceEditResponseBody =
       -- logging or to provide a suitable error for a request that
       -- triggered the edit.
     , _failureReason :: Maybe Text
+      -- | Depending on the client's failure handling strategy `failedChange`
+      -- might contain the index of the change that failed. This property is
+      -- only available if the client signals a `failureHandling` strategy
+      -- in its client capabilities.
+    , _failedChange :: Maybe Word32
```

Comment:
This is a not related change, maybe it worths to note it as a separate commit or in another way

Reply:
Oops thanks, just a small thing I noticed we missed!